### PR TITLE
S02 log in pop up

### PR DESCRIPTION
**Pull request by pagouillart**

### CODE REVIEW

**Review by Cyrildab — APPROVED** [review]

**Review by Romlion — APPROVED** [review]

**Review by LucasMerino10 — APPROVED** [review]

**Review by Rayane0823 — APPROVED** [review]

File: frontend/src/components/LogInPopup/LogInPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <Dialog.Content className="dialogContent">
+          <Dialog.Title className="dialogTitle">
+            Connexion à mon espace
+          </Dialog.Title>
+          <Dialog.Description className="dialogDescription">
+            Déjà inscrit ? Connectez-vous.
+          </Dialog.Description>
+          <fieldset className="fieldset">
+            <label className="label" htmlFor="UserName">
+              Nom d'utilisateur
```
Comment (Yoann-Fortin) [inline frontend/src/components/LogInPopup/LogInPopup.jsx]: It's confusing to ask for the name but wait for the email.

File: frontend/src/components/LogInPopup/LogInPopup.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            </label>
+            <input className="input" id="UserName" placeholder="email" />
```
Comment (Yoann-Fortin) [inline frontend/src/components/LogInPopup/LogInPopup.jsx]: Input type is missing

File: frontend/src/components/LogInPopup/LogInPopup.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            <Dialog.Close asChild>
+              <button type="button" className="button LogIn PopUP">
+                Connexion
+              </button>
+            </Dialog.Close>
+            <Dialog.Close asChild>
+              <button
+                type="button"
+                className="closingButton"
+                aria-label="Close"
```
Comment (Yoann-Fortin) [inline frontend/src/components/LogInPopup/LogInPopup.jsx]: Your site is in French, so no English

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/LogInPopup/LogInPopup.jsx (hunk 1 above)
Comment (pagouillart) [reply frontend/src/components/LogInPopup/LogInPopup.jsx]: solved

File: frontend/src/components/LogInPopup/LogInPopup.jsx (hunk 2 above)
Comment (pagouillart) [reply frontend/src/components/LogInPopup/LogInPopup.jsx]: solved

File: frontend/src/components/LogInPopup/LogInPopup.jsx (hunk 3 above)
Comment (pagouillart) [reply frontend/src/components/LogInPopup/LogInPopup.jsx]: solved

**Review by Romlion — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**